### DISCUSSION TITLE
Update brew to the latest version under the Mac of M2 chip and install shivammathur/php/ php@7.4 Throwing an undefined method error

### DISCUSSION THREAD

**Discussion by 877371623** [shivammathur/homebrew-php#2354] [Q&A / Help]
My system uses the Mac M2 chip. Firstly, I update brew to the latest version 4.2.9, and then run brew install shivammathur/php/ php@7.4 Throwing shivammathur/php when installing PHP7.4/ php@7.4 : undefined method ` plistoptions' for Formula:: FormulaNamespaceff771b7a9f8893f87b5f9663cc33b1c8cd7e0ec48952ba3a25010099bd6b5b34:: PhpAT74: There is an error with Class. I tried uninstalling and reinstalling Brew, but it still did not solve the problem. I returned Brew to version 3.6 without this error, but there are other compatibility issues that prompt me to upgrade Brew to the latest version

<img width="1297" alt="WeChat74d5d33a8f29ff7e929f1a08cb33f1dc" src="https://github.com/shivammathur/homebrew-php/assets/68764028/9202c372-0ce5-430a-ae73-3ba5e618a32a">

**Comment by shivammathur** [comment]
`plist_options` was removed last year in https://github.com/shivammathur/homebrew-php/commit/e7da7404b6bf83e2d62e6f79e603eb16d4f43030.

Please update the tap as well.

```
HOMEBREW_DEVELOPER=1 brew untap shivammathur/php
brew tap shivammathur/php
```

Then try again.
